### PR TITLE
Chore/docker Created dockerfile to dockerize monika and Makefile to make docker build and push easier.

### PR DESCRIPTION
**Pull request by newm4n**
- Created Dockerfile, to dockerize monika
- Created Makefile, to simplify build.

In your development you can do:

- make doker <-- to build monika docker image locally
- make docker-run <-- build and run the docker image in your locally
- make docker-stop <-- stop the monika docker that started using docker-run
- make docker-push <-- push the docker image into docherhub.io. Remember to change the VERSION first.

### CODE REVIEW

**Review by armedi — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -43,6 +43,15 @@ For more information, please refer to the detailed documentations below.
...
 
+## Run Monika using Docker
+
+Obtain a copy of your configuration file `config.json` into the current directory.
+From the same directoy, you can run:
+
+```
+docker run --name monika -v ${PWD}/config.json:/config/config.json --detach hyperjump/monika:latest
+```
+
```
Comment (nicnocquee) [inline README.md:54]: Maybe also shows the command to view the logs from the docker container.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,28 @@
+GOPATH=$(shell go env GOPATH)
```
Comment (nicnocquee) [inline Makefile:1]: Is this used?

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+IMAGE_REGISTRY=dockerhub
+IMAGE_NAMESPACE ?= hyperjump
+IMAGE_NAME ?= monika
+CURRENT_PATH=$(shell pwd)
+COMMIT_ID ?= $(shell git rev-parse --short HEAD)
+VERSION=v1.1.0
+GO111MODULE=on
```
Comment (nicnocquee) [inline Makefile:8]: Is this used?

**Review by nicnocquee — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (newm4n) [reply README.md:54]: If they use docker, they should've know how to see docker log.

File: Makefile (hunk 3 above)
Comment (newm4n) [reply Makefile:8]: removed unnecessary

File: Makefile (hunk 2 above)
Comment (newm4n) [reply Makefile:1]: removed

File: README.md (hunk 1 above)
Comment (nicnocquee) [reply README.md:54]: Please still add it. It's not a difficult thing to do, and if they forgot how to do it, we can save them time. 😃

**Review by nicnocquee — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (newm4n) [reply README.md:54]: The thing is, its a docker subject. and expect the user to know how to operate docker prior interested to use monika docker, if not, they should've not used docker.

docker logs <container name> is so trivial, that you dont have to explain to user. its like explaining how to install npm to develop node app. 

if we entertain this, there will be question "how to stop ?" "how to remove image?" "how to run in non detach?" 
'how to auto start when docker start?"

**Review by nicnocquee — APPROVED** [review]

**Pull request merged**